bot.user.js

```diff
@@ -434,9 +431,9 @@ var bot = (function() {
         },
 
         // Sorting function for food, from property 'clusterCount'
-		sortFood: function(a, b) {
-			return (a.clusterScore == b.clusterScore ? 0 : a.clusterScore / a.distance  >  b.clusterScore / b.distance  ? -1 : 1);
-		},
+        sortFood: function(a, b) {
```

Comment:
Every function in those modules are named like this, should they all be changed then?